content/guides/references/best-practices.md

```diff
@@ -414,11 +414,21 @@ How to solve this:
 
 Let's imagine the following test that is filling out the form.
 
-```javascript
+:::cypress-visit-mount-test-example
+
+```js
+cy.visit('/users/new')
+```
+
+```js
+mount(<MyComponent />)
+```
+
+```js
 // an example of what NOT TO DO
 describe('my form', () => {
   it('visits the form', () => {
```

Comment:
When the component is being mounted, it's not still visiting the form. The entire `it()` should probably be replaced for CT. Can we also call the component something like `UserForm`?

Reply:
Good catch.